src/builder/QuarkBuilder.sol

```diff
@@ -16,16 +16,14 @@ contract QuarkBuilder {
     /* ===== Constants ===== */
 
     string constant VERSION = "1.0.0";
-    uint256 constant MAX_BRIDGE_ACTION = 1;
 
     /* ===== Custom Errors ===== */
 
     error AssetPositionNotFound();
-    error FundsUnavailable(uint256 requiredAmount, uint256 actualAmount, uint256 missingAmount);
+    error FundsUnavailable(string assetSymbol, uint256 requiredAmount, uint256 actualAmount, uint256 missingAmount);
```

Comment:
I know this wasn't introduced in your PR, but `missingAmount` seems like an unnecessary argument.

It's always `requiredAmount - actualAmount`, right?

Reply:
Yeah, I had the same thought. I can remove it

Reply:
Done